tests/tracer/test_ext.py

```diff
@@ -21,7 +25,154 @@ def _updateenv(monkeypatch, env):
         monkeypatch.setenv(str(k), str(v))
 
 
+@pytest.fixture
+def git_repo(tmpdir):
+    cwd = str(tmpdir)
+    subprocess.check_output("git init", cwd=cwd, shell=True)
+    subprocess.check_output('git remote add origin "git@github.com:test-repo-url.git"', cwd=cwd, shell=True)
+    subprocess.check_output('git config --local user.name "John Doe"', cwd=cwd, shell=True)
+    subprocess.check_output('git config --local user.email "john@doe.com"', cwd=cwd, shell=True)
+    subprocess.check_output("touch tmp.py", cwd=cwd, shell=True)
+    subprocess.check_output("git add tmp.py", cwd=cwd, shell=True)
+    subprocess.check_output(
+        'git commit --date="2021-01-19T09:24:53-0400" -m "this is a commit msg" --no-edit', cwd=cwd, shell=True
+    )
+    subprocess.check_output(
+        'GIT_COMMITTER_DATE="2021-01-20T04:37:21-0400" git -c user.name="Jane Doe" -c user.email="jane@doe.com" commit '
```

Comment:
do you need `-c user.name....` if it is set in the local config?

Reply:
wait, this does an amend, are we not able to do this as part of the previous command?

Reply:
Correct me if I'm wrong and there's a better way, but I wanted to set a different user as the committer and this was the only way I could overwrite the committer without changing the author 😆

Reply:
committer vs author?

Reply:
oh, no idea.... 

From `git commit --help`

```
--author=<author>
           Override the commit author. Specify an explicit author using the standard A U Thor <author@example.com> format. Otherwise <author> is assumed to be a
           pattern and is used to search for an existing commit by that author (i.e. rev-list --all -i --author=<author>); the commit author is then copied from
           the first such commit found.
```

?

Reply:
haha that works much cleaner 😢 why didn't I see that earlier!?